src/skills-builder/skills-builder-modal/view-results/RecommendationCard.jsx

```diff
@@ -30,18 +36,15 @@ const RecommendationCard = ({
     >
       <Card.ImageCap
         src={cardImageUrl}
-        logoSrc={logoImageUrl}
+        logoSrc={schoolLogoUrl}
         fallbackSrc={cardImageCapFallbackSrc}
         fallbackLogoSrc={cardImageCapFallbackSrc}
       />
       <Card.Header title={title} size="sm" />
       <Card.Section>
-        {partner.map((orgName, index) => (
-          // eslint-disable-next-line react/no-array-index-key
-          <Chip key={index} className="chip-max-width">
-            {orgName}
-          </Chip>
-        ))}
+        <Chip className="chip-max-width">
```

Comment:
Is there really ever more than one partner in the data? There is only ever a single override.
Do we need a more complex approach that uses the singular override OR the array of partners?

Reply:
After further consideration, since we are only grabbing one name at 
https://github.com/edx/frontend-app-skills/blob/52a697820d340907a5c1a8e37d3589b5ef62c57c/src/skills-builder/skills-builder-modal/view-results/RecommendationCard.jsx#L23

We probably want to keep that pattern and just grab the first one in the `partner` list